src/go/util/string.go

```diff
@@ -19,3 +21,32 @@ func RandomString(n int) string {
 
 	return string(b)
 }
+
+// Credit: https://github.com/go-yaml/yaml/issues/100#issuecomment-901604971
+//
+// Allow yaml parsing for an array of strings or a single inlined element:
+//
+//	field: value
+//
+//	  or
+//
+//	field:
+//	  - value1
+//	  - value2
+type StringArray []string
```

Comment:
Um... no. :joy:

Let's **strongly** consider naming this `StringSlice` instead please!

Reply:
How about `type A []string`